Orange/evaluation/scoring.py

```diff
@@ -249,6 +249,14 @@ def compute_score(self, results):
         return np.sqrt(MSE(results))
 
 
+class CVRMSE(Score):
+    """
+    Coefficient of variation of the RMSE
```

Comment:
I like the idea of adding more (not dozens maybe) scores. We had some requests like sensitivity and selectivity from scientists who are interested in using the infrared addon. Also, it would be nice, if the scores are displayed in an easily copy-pasteable table...